docker-for-mac/release-notes.md

```diff
@@ -58,6 +60,9 @@ Docker Desktop Community 2.1.0.0 contains the following experimental features.
 
 ### Bug fixes and minor changes
 
+* Docker Desktop now allows users to expose privileged UDP ports. [docker/for-mac#3775](https://github.com/docker/for-mac/issues/3775)
+* Fixed an issue where running some Docker commands can fail if you are not using Credential Helpers. [docker/for-mac#3785](https://github.com/docker/for-mac/issues/3785)
+* Updated Hyperkit to version [0.20190802](https://github.com/moby/hyperkit/releases/tag/v0.20190802).
```

Comment:
Should this one be in the upgrades section?

Reply:
Yes, that's right. I'll move this to the upgrades section.